manager/allocator/network.go

```diff
@@ -948,15 +975,18 @@ func (a *Allocator) procUnallocatedServices(ctx context.Context) {
 	}
 }
 
-func (a *Allocator) procUnallocatedTasksNetwork(ctx context.Context) {
-	nc := a.netCtx
-	allocatedTasks := make([]*api.Task, 0, len(nc.unallocatedTasks))
+func (a *Allocator) procTasksNetwork(ctx context.Context, toAllocate map[string]*api.Task, quiet bool) {
```

Comment:
If working on the nc retrieved from the context is equivalent, would it make sense to write this method as 
```
func (a *Allocator) procTasksNetwork(ctx context.Context, onRetryInterval bool) {
    nc := a.netCtx
    quiet := false
    toAllocate := nc.pendingTasks
    if onRetryInterval {
        toAllocate = nc.unallocatedTasks
        quiet = true
    }
...
```